src/CatCard/styles.js

```diff
@@ -0,0 +1,38 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  catCard: {
+    borderColor: 'grey',
```

Comment:
All colors better to move to constants file so in your we will have good style system